### DISCUSSION TITLE
Kurze Anleitung Git

### DISCUSSION THREAD

**Discussion by MehmetCagferoglu** [irgroup-classrooms/dis08-2021#20] [Q&A]
Hallo Zusammen,

ich hab noch einige kleine Probleme mit Git.

Könnte hier jemand eventuell  eine Anleitung erstellen wie ich das Assigment richtig bearbeite und speichere.

Vielen Dank

**Comment by HaakFab** [comment]
Guten Tag,
sollten Sie sich auf "Exercise 3: Git" aus dem ersten Assignment beziehen, machen Sie sich keine Sorge– das wird in der Vorlesung dazu (diese Woche) und nächste Woche in der Übung behandelt.

VG,
Fabian Haak
